build/configs/rtl8721csm/Make.defs

```diff
@@ -171,6 +171,14 @@ ifeq ($(CONFIG_HAVE_CXX),y)
   CXXFLAGS += -pedantic -D_DEBUG -D_LIBCPP_BUILD_STATIC -ffunction-sections -fdata-sections -D__GLIBCXX__
 endif
 
+ifeq ($(CONFIG_EXTERNAL_TFMICRO),y)
+CXXFLAGS += -DTF_LITE_STATIC_MEMORY -DNDEBUG -g
+CFLAGS   += -DTF_LITE_STATIC_MEMORY -DNDEBUG -g
+CXXFLAGS += -I$(TOPDIR)/../external/tfmicro
+CXXFLAGS += -I$(TOPDIR)/../external/tfmicro/third_party
+CXXFLAGS += -I$(TOPDIR)/../external/tfmicro/third_party/gemmlowp
+endif
```

Comment:
Are they necessary here? not in the folder which uses TFMICRO like aifw?

Reply:
these are included here, as it is required in build of framework/src/AIFW & apps/examples/aifw_test also.